bonding-rose.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#bonding-rose
+=end
+
+custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager])
+
+class BondingRose
+  include DRC
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    @settings = get_settings
+
+    unless DRStats.cleric?
+      echo '***MUST BE A CLERIC***'
+      exit
+    end
+
+    make_rose
+  end
+
+  def make_rose
+    temp_settings = @settings
+    temp_settings.hometown = 'Crossing'
+    ensure_copper_on_hand(1000, temp_settings)
+    EquipmentManager.new.empty_hands
+    unless exists?('mortar')
+      order_item(8860, 1)
+      bput('stow mortar', 'You put')
+    end
+    unless exists?('pestle')
+      order_item(8860, 3)
+      bput('stow pestle', 'You put')
+    end
+    unless exists?('mixing stick')
+      order_item(8860, 5)
+      bput('stow mix stick', 'You put')
+    end
+    unless exists?('small jar')
+      buy_item(19076, 'small jar')
+      bput('get oil in my small jar', 'You get')
+      bput('drop my oil', 'You drop')
+      bput('stow my small jar', 'You put')
+    end
+    buy_item(19076, 'oil')
+    fput('prep bless')
+    pause 8
+    fput('cast my oil')
+    bput('put my oil in my small jar', 'You put')
+    buy_item(19076, 'dracon crystals')
+    bput('put my crystals in my mortar', 'You put', 'The mortar already has')
+    bput('get my mortar', 'You get')
+    bput('get my pestle', 'You get')
+    while bput('crush crystals in my mortar with my pestle', 'into some dracon powder', 'with your pestle') == 'with your pestle'
+      pause 1
+    end
+    EquipmentManager.new.empty_hands
+    bput('get powder from my mortar', 'You get')
+    bput('break my powder', 'You break')
+    bput('drop my powder', 'You drop')
+    bput('break my powder', 'You break')
+    bput('drop my powder', 'You drop')
+#   This leaves the 3 parts needed in your hand
+    bput('put my powder in my small jar', 'You put')
+    EquipmentManager.new.empty_hands
+
+    walk_to(1977)
+    bput('look bough', 'The bough sways')
+    move('go path')
+    walk_to(12133)
+    case bput('get petal', 'You gather', 'You decide to leave')
+    when 'You decide to leave'
+      echo('You have gotten petals too recently, retry in 2 hours.')
+      exit
+    end
+    bput('put my petal in my mortar', 'You put')
```

Comment:
Can you put the petal in the mortar before `get`ting the mortar (line 79)?

Reply:
sure can, it goes in the mortar in your stow container

Reply:
Neat, I wasn't aware of that. Thank you.